doc/test-framework/scorecard.md

```diff
@@ -386,16 +386,162 @@ Example of a valid JSON output:
 **NOTE:** The `ScorecardOutput.Log` field is only intended to be used to log the scorecard's output and the scorecard will ignore that field if a plugin provides it.
 To add logs to the main `ScorecardOuput.Log` field, a plugin can output the logs to `stderr`.
 
-## Running the scorecard with a deployed CSV
-
-The scorecard can be run using only a [Cluster Service Version (CSV)][olm-csv], providing a way to test cluster-ready and non-SDK operators.
+## Running the scorecard with an OLM-managed operator
```

Comment:
nit: required to fix/update the index. I would recommend do it automatically by using doctoc and them removing its comments.

Reply:
Are we doing this for all documentation now?

Reply:
There was some discussion a while back about how best to integrate doctoc given that it's installed with `npm` (I think). My concern was adding a non-go, non-binary dependency to our build and CI scripts.

Reply:
At the very least in this PR, we can manually fix the DocToc section to match the new header.

Longer-term, @asmacdo is interested in a much larger refactoring and re-homing of our docs to a readthedocs or gitbook site, so I'd rather we not try to make any major changes to our current docs automation (with the exception of @theishshah CLI docs generation)